src/preamble.js

```diff
@@ -608,14 +608,14 @@ function instrumentWasmTableWithAbort() {
 #endif
 
 var wasmBinaryFile;
-#if EXPORT_ES6 && USE_ES6_IMPORT_META && !SINGLE_FILE
+#if EXPORT_ES6 && USE_ES6_IMPORT_META && !SINGLE_FILE && !AUDIO_WORKLET // `URL` is undefined in Audio Worklet scope, so we cannot use `new URL()`.
 if (Module['locateFile']) {
```

Comment:
This ifdef seems to only affect `locateFile`, how does that related to `new URL()`?

Reply:
Hello, thanks for reviewing.

The idea was to show that this ifdef and the next ifdef are linked/were changed for the same reason.

But it seems having it worded like that might be worse than simply not having the comment there at all. I'll remove it.

Reply:
This change looks not removed yet?